website/src/components/sidebar.css.ts

```diff
@@ -38,31 +63,42 @@ export const header = style([
   },
 ])
 
-export const desktopNav = style([
-  navButton,
+export const openNavButton = style([
   bgColor,
+  padding(hspace.halfEdge),
   {
     "@media": {
       [mediaQueries.medium]: {
-        display: "block",
-        position: "absolute",
-        color: colors.body,
-        borderRadius: radii.medium,
+        width: `calc(${hsize.xsmall} / 2)`,
+        height: `calc(${hsize.xsmall} / 2)`,
```

Comment:
If you need this size, add a constant to `size` or `hsize` and `vsize`. Only use `calc` for values that change at runtime or are impossible to calculate at build time.

Reply:
would it be fine just to just use "3rem" (the desired size) rather than adding it to the list of constants? I'm just worried about adding too many different constants to `hsize` which may make it cluttered